### ISSUE TITLE
[enhancement] for macos

### ISSUE THREAD

**Issue by Leo-Lionni** [topxeq/gotx#1]
could this project  support  Macos? i preferrly meet gotx  with macbook

**Comment by topxeq** [comment]
Yes, you can get the latest Mac version from the website of Gox/Gotx [here](http://topget.org/gox.html).
